chart/templates/mayastor/agents/core/agent-core-deployment.yaml

```diff
@@ -25,6 +25,9 @@ spec:
         {{- include "base_pull_secrets" . }}
       initContainers:
         {{- include "base_init_core_containers" . }}
+      {{- if .Values.priorityClassName }}
+      priorityClassName: {{ .Values.priorityClassName }}
```

Comment:
this is already defined a few lines below

Reply:
As @tiagolobocastro said this should probably come from `values.yaml` for all the components. And specifically for the control-plane components we should default to `system-cluster-critical`, if the value is not set.

Reply:
Argh yes I did not see this one had it hardcoded, assumed that it wouldn't due to all the others not also having it, will amend, thanks for checking 👍